CHANGELOG.md

```diff
@@ -2,6 +2,39 @@
 
 This document outlines major changes between releases.
 
+## 0.106.2 "K" (13 Jun 2024)
```

Comment:
It's hard this time, I tried to think about something release-related.

Reply:
I've only got two other "proper" possibilities ("Karstification" already taken):
 * Kation
 * Keratinization

Reply:
> Keratinization

That was my option too, so decided.